com.ibm.wala.core/src/com/ibm/wala/classLoader/IMethod.java

```diff
@@ -43,7 +43,14 @@
   boolean isNative();
 
   /**
-   * Did someone synthesize this method? (As opposed to reading it from a class file)
+   * Is this method created by WALA?
```

Comment:
Can we rephrase this "Is the implementation of this method a model generated by WALA?"  I think in many cases the method exists in the original bytecode, we just hijack the implementation.  Saying the method is created by WALA is a bit ambiguous.